### PR TITLE
[net-libs/telepathy-logger-qt] add 0.9.80 release

### PR DESCRIPTION
**Pull request by karolherbst**
Package-Manager: portage-2.2.18

### CODE REVIEW

File: net-libs/telepathy-logger-qt/telepathy-logger-qt-0.9.80.ebuild [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+if [[ ${PV} == *9999 ]]; then
+	KEYWORDS=""
+else
+	SRC_URI="mirror://kde/unstable/kde-telepathy/${PV}/src/${P}.tar.xz"
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="LGPL-2.1"
+SLOT="5"
```
Comment (johu) [inline net-libs/telepathy-logger-qt/telepathy-logger-qt-0.9.80.ebuild]: SLOT 5 is already set by eclass. Sould be fixed in live as well.

File: net-libs/telepathy-logger-qt/telepathy-logger-qt-0.9.80.ebuild [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 )
+inherit kde5 python-any-r1
+
+DESCRIPTION="Qt bindings for the Telepathy logger"
+HOMEPAGE="https://projects.kde.org/projects/extragear/network/telepathy/telepathy-logger-qt"
+
+if [[ ${PV} == *9999 ]]; then
```
Comment (johu) [inline net-libs/telepathy-logger-qt/telepathy-logger-qt-0.9.80.ebuild]: ${KDE_BUILD_TYPE} = live

**Comment by karolherbst** [comment]
@johu done

**Comment by johu** [comment]
Thanks. This is rebased onto kde overlay ,master with changed commit message "[net-libs/telepathy-logger-qt] Version bump"

**Pull request closed without merging**